index.html

```diff
@@ -5,24 +5,151 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Celine Vogel Personal Website</title>
+<link rel="stylesheet" href="styles.css">
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=Roboto+Mono:wght@300&display=swap');
+  </style> 
 </head>
 <body>
 <header>
     <h1>Celine Vogel</h1>
-    <nav>
-        <a href="#contact">Contact</a>
-        <a href="#biography">Biography</a>
+
+    <br>
```

Comment:
Not sure here, but I would not use `<br>` for styling html. Instead I would solve this with CSS.

Reply:
Your are right, I am going to change it 🗡️